config/jest/transform.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+const babelDev = require('../babel.dev');
+const babelJest = require('babel-jest');
+
+module.exports = babelJest.createTransformer(babelDev);
```

Comment:
this seems like the nicest way to ensure low-maintenance of this script as we add more stuff to babel-jest.